pkg/manifests/ext_proc.yaml

```diff
@@ -94,7 +94,6 @@ spec:
         request:
```

Comment:
I see we have files in https://github.com/kubernetes-sigs/gateway-api-inference-extension/tree/main/examples/poc/manifests/gateway too, are those still relevant?

Reply:
Removed them. They are already outdated.